recipe/meta.yaml

```diff
@@ -9,21 +9,21 @@ source:
   sha256: 73cc4d115b96f79c7d77c1c7f7a0a8d4c57860d1041df407dd1aae7f07a77fd7
 
 build:
-  # Uses 2to3 and cannot be noarch.
-  number: 0
+  number: 1
 
 outputs:
   - name: beautifulsoup4
+    build:
+      noarch: python
     script: build_base.bat  # [win]
```

Comment:
Should we remove this too?

Reply:
It's fine to keep there for those who want to build the noarch: python package from windows.